backends/dpdk/dpdkAsmOpt.h

```diff
@@ -173,6 +173,125 @@ class ValidateTableKeys : public Inspector {
     int getFieldSizeBits(const IR::Type *field_type);
 };
 
+// This pass shorten the Identifier length
+class ShortenTokenLength : public Transform {
+    ordered_map<cstring, cstring> newNameMap;
+    ordered_map<cstring, cstring> &origNameMap;
+    static size_t count;
+    // Currently Dpdk allows Identifier of 63 char long or less
```

Comment:
this is just a silly restriction, since I suspect the identifiers do not have a runtime representation.
wouldn't it be easier to fix this in the DPDK spec/implementation?
we're no longer in the 1970s when every byte counted.